docs/platforms/react-native/session-replay/index.mdx

```diff
@@ -44,6 +44,24 @@ Sentry.init({
 });
 ```
 
+```javascript {tabTitle:Browser}
+import * as Sentry from '@sentry/react-native';
+import { Platform } from 'react-native';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: (integrations) => {
+    if (Platform.OS === 'web') {
+      integrations.push(Sentry.browserReplayIntegration());
+    }
+    integrations.push(Sentry.mobileReplayIntegration());
```

Comment:
should this be in an else block?

Reply:
The mobile replay can be added on any platform. So for simplicity it doesn't have to be.